styles.css

```diff
@@ -0,0 +1,541 @@
+.logo {
+  visibility: hidden;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  width: 57%;
+  gap: 35px;
+}
+
+.nav-list a {
+  color: var(--Primary);
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.nav-list li a:hover {
+  color: var(--Secondary);
+}
+
+.contant {
+  flex-direction: column;
+  gap: 24px;
+  justify-content: center;
+  height: 100vh;
+}
+
+h1 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Primary);
+}
+
+h2 {
+  font-weight: 1300;
+  font-size: 70px;
+  line-height: 70px;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: var(--Secondary);
+}
+
+p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: var(--Primary);
+  flex: none;
+  flex-grow: 0;
+  max-width: 750px;
+}
+
+.social {
+  gap: 20px;
+  margin-top: 3%;
+}
+
+.next {
+  margin-top: 5%;
+  display: flex;
+}
+
+.head {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: var(--Primary);
+}
+
+.img-multi {
+  background-image: url(assest/img/recent/holder.png);
+  background-size: cover;
+  height: 260px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.cards {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+}
+
+.card {
+  height: 474px;
+  width: 327px;
+  flex-direction: column;
+  border: 1px solid #ebf0ee;
+  border-radius: 10px;
+}
+
+.text {
+  margin-bottom: 25px;
+  flex-direction: column;
+}
+
+h3 {
+  margin-bottom: 5%;
+}
+
+.tags ul {
+  flex-direction: row;
+  list-style: none;
+  column-gap: 5px;
+  padding: 5px;
+  margin-bottom: 20px;
+}
+
+.tag {
+  background: #ebf0ee;
+  padding: 5px;
+  border: 1px solid #fff;
+  margin-bottom: 5px;
+  color: #000;
+  border-radius: 5px;
+  font-size: 12px;
+  line-height: 16px;
+  font-weight: 600;
+}
+
+.see-project {
+  background: var(--Secondary);
+  border: none;
+  color: #fff;
+  border-radius: 5px;
+  padding: 5px 12px;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+}
+
+.see-project:hover {
+  background-color: #36b37f;
+  cursor: pointer;
+}
+
+.contant-1 {
+  flex-direction: column;
+  position: relative;
+  gap: 24px;
+  text-align: center;
+}
+
+.get {
+  background: #36b37f;
+  border: none;
+  padding: 9px 19px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  border-radius: 5px;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  color: white;
+}
+
+.tager {
+  display: grid;
+  align-items: center;
+  gap: 1.5rem;
+  padding: 1.5rem 0;
+  margin: 0 1rem;
+  grid-template-columns: auto auto auto;
+  justify-content: center;
+}
+
+.card-about {
+  background-color: #ebf0ee;
+  height: 352px;
+  width: 369px;
+  flex-direction: column;
+  border: 1px solid black;
+  border-radius: 10px;
+}
```

Comment:
- [ ] For each of the cards in the _**About Myself**_ section, you need to use the `margin` values from the Figma template for the left and right side of each card in this section. Also, the `border` of each card is `1px solid #C1C7D0` 😎 


### Your Design

![image](https://user-images.githubusercontent.com/77106962/219493243-70da4239-1ddf-4142-8196-c68797bfb855.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/219493336-e605e7ab-fe0c-42be-96aa-fa8b41bff338.png)